src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -3371,16 +3371,25 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
     }
 
     async getVersion(): Promise<string> {
-        const result: [{ version: string }] = await this.query(
+        const result = await this.query(
             `SELECT VERSION() AS \`version\``,
```

Comment:
Let's simplify. As discussed in the issue, `SELECT version()` would work on all versions:

```suggestion
        const result: [{ "version()": string }] = await this.query(
            `SELECT version()`,
```